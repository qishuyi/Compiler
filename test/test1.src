if 10 <= 8 then 8 / 2 + 6 else 2 * 6 - 10