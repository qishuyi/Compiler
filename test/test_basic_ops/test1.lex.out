['if' (1 : 1),10 (2 : 9),<= (2 : 12),8 (2 : 15),'then' (3 : 1),8 (4 : 9),/ (4 : 11),2 (4 : 13),+ (4 : 15),6 (4 : 17),'else' (5 : 1),2 (6 : 9),* (6 : 11),6 (6 : 13),- (6 : 15),10 (6 : 17)]